src/components/categories.css

```diff
@@ -1,6 +1,11 @@
 .menu {
   display: flex;
-  padding: 1rem;
+  width: 100%;
+  padding-left: 8rem;
+  padding-right: 8rem;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
```

Comment:
- [ ] I suggest you to use the shorthand version for this padding, it will make your code shorter and easier to read 🧠 
You can achieve that this way : `padding: 1rem 8rem;`